pennylane/tensorbox/tensorbox.py

```diff
@@ -195,13 +265,20 @@ def expand_dims(self, axis):
                 dimensions should be inserted
         """
 
+    @property
+    @abc.abstractmethod
+    def interface(self):
+        """str, None: The package that the :class:`.TensorBox` class

```

Comment:
Yep! This will show up as an 'attribute' in the documentation.

Sphinx will interpret this exactly the same as if we had a `Returns:` block, but it avoids the redundant description of the returns argument.

I wonder if this also works for small methods?